packages/table/src/layers/focusCell.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+export interface IFocusedCellCoordinates {
```

Comment:
Hm...

So at some point I need to add a third number (or property?) to this to identify which selected region the focused cell belongs to. I could make a type that extends the ICellCoordinates type -- but I am still going to need that third property, and another name for the thing.